### DISCUSSION TITLE
Pagination should include component tokens to adjust the placement of showTotal props

### DISCUSSION THREAD

**Discussion by zxlvera** [ant-design/ant-design#49077] [Ideas]
From this where `showTotal` is right next to page navigation:

![image](https://github.com/ant-design/ant-design/assets/17290983/5319b014-8fef-4974-a91c-56ce378a128a)

to this, where `showTotal` and page navigation have space in between:

![image](https://github.com/ant-design/ant-design/assets/17290983/e556b439-9035-48fc-adbe-b57c9fdd7efc)

```
const TableContainer = styled.div`
  width: 100%;
  padding-top: 16px !important;

  ul.ant-pagination > li.ant-pagination-total-text {
    margin-right: auto;
  }
`;
```

**Comment by nouman2075** [comment]
Using flex, you can position the `showTotal` text to the right
```
const TableContainer = styled.div`
  width: 100%;
  padding-top: 16px !important;

  ul.ant-pagination {
    display: flex;
  }

  ul.ant-pagination > li.ant-pagination-total-text {
    flex-grow: 1;
  }
`;
```
